programs/wen_new_standard/src/instructions/group/create.rs

```diff
@@ -107,13 +109,13 @@ impl<'info> CreateGroupAccount<'info> {
         Ok(())
     }
 
-    fn update_mint_authority(&self, manager_auth: Pubkey) -> Result<()> {
+    fn update_mint_authority(&self) -> Result<()> {
         let cpi_accounts = SetAuthority {
             current_authority: self.authority.to_account_info(),
             account_or_mint: self.mint.to_account_info(),
         };
         let cpi_ctx = CpiContext::new(self.token_program.to_account_info(), cpi_accounts);
-        set_authority(cpi_ctx, AuthorityType::MintTokens, Some(manager_auth))?;
```

Comment:
why drop this? We need this authority to be able to add Group and Member TLV once its live on mainnet.

Reply:
Will add it back in a different PR with burn, delegate and freeze